src/PowerSystems.jl

```diff
@@ -147,6 +147,7 @@ export SteamTurbineGov1
 export Converter
 export AverageConverter
 export RenewableEnergyConverterTypeA
```

Comment:
We need to add a note here for 2.0 that this should become `RenewableEnergyCurrentConverterTypeA`